code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -156,6 +156,15 @@
 	///Generic flags
 	var/simple_mob_flags = NONE
 
+//SKYRAT EDIT-: ALL SIMPLE MOBS NOW WARNED
+/mob/living/simple_animal/Login()
+	. = ..()
+	if(!. || !client)
+		return FALSE
+	to_chat(src, "<b>As a simple animal you are nothing but a feral creature with some small modicum of intelligence.</b>")
+	to_chat(src, "<span class='warning'>This also means if you're engaging in combat you should refrain from mercing people fully. Play not to win but to create a challenge. You're there to replace AI, make others enjoy the situation as well. If your simple mob is not above simple or 'mute' intelligence, using structures such as welding tanks/canisters/boxes to hinder your opponent is entirely forbidden. Do not do this.</span>")
```

Comment:
```suggestion
	var/warning_string = CONFIG_GET(string/pc_mob_text)
	if(warning_string)
		to_chat(src, span_warning(warning_string))
```
Should probably bind this to config, also, span procs

Reply:
span proc man bad and also I was not aware they existed

sidenote I dont think thats an actual value thats been set

Reply:
nevermind it exists

Reply:
Span procs are good cause it prevents <span class='notice'>You do a thing/span>"

But yeah here's the config https://github.com/Skyrat-SS13/Skyrat-tg/blob/13f617c968dc64a0ccf9bea2dca65753b382ce17/modular_skyrat/master_files/code/modules/mob/living/simple_animal/simple_animal.dm#L1